Corona-Warn-App/src/main/res/layout/recovery_certificate_card.xml

```diff
@@ -12,8 +12,8 @@
         android:id="@+id/certificate_bg"
         android:layout_width="88dp"
         android:layout_height="95dp"
-        android:layout_marginStart="16dp"
```

Comment:
If I recall correctly, we decided at some point within the android team to rather use explicit dimension - that is 16dp over @dimen/spacing_small for better tooling support within Android Studio and easier readability.